oaknational/Oak-Web-Application
# feat: lesson resource graphics

## Description

- Lesson resource graphic for lesson listing page item.

## Issue(s)

Fixes #1050 

## How to test 

1. Go to  storybook - element
3. Click on _______
4. You should see _______

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
